capstone/src/main/java/com/google/sps/servlets/VoterServlet.java

```diff
@@ -36,21 +37,21 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     URLConnection connection = new URL(url + "?" + query).openConnection();
     connection.setRequestProperty("Accept-Charset", charset);
     JSONObject electionJsonObject = getJSONObject(connection);
-    String electionName = getElectionDay(electionJsonObject);
-    String electionDay = getElectionName(electionJsonObject);
-    request.setAttribute("name", electionName);
-    request.setAttribute("electionDay", electionDay);
+    String electionName = getElectionName(electionJsonObject);
+    String electionDay = getElectionDay(electionJsonObject);
+    VotingEvent event = new VotingEvent(electionName, electionDay);
```

Comment:
I like what you did with the helper methods, however they seem a bit too granular.
 I think you can just create one:
```private VotingEvent getVotingEvent(JSONObject electionJsonObject){...}```
This method can just call the getters and return the complete voting event object.